src/elements/content-sidebar/ActivitySidebarFilter.js

```diff
@@ -9,29 +9,51 @@ import { FormattedMessage } from 'react-intl';
 import DropdownMenu, { MenuToggle } from '../../components/dropdown-menu';
 import PlainButton from '../../components/plain-button';
 import messages from './messages';
-import { COMMENT_STATUS_OPEN } from '../../constants';
+import {
+    ACTIVITY_FEED_ITEM_ALL,
+    ACTIVITY_FEED_ITEM_OPEN,
+    ACTIVITY_FEED_ITEM_RESOLVED,
+    ACTIVITY_FEED_ITEM_TASKS,
+    ACTIVITY_FILTER_TITLE_ALL_COMMENTS,
+    COMMENT_STATUS_OPEN,
+    COMMENT_STATUS_RESOLVED,
+} from '../../constants';
 import { Menu, SelectMenuItem } from '../../components/menu';
-import type { FeedItemStatus } from '../../common/types/feed';
+import type { ActivityFilterAllTitle, ActivityFilterStatus } from '../../common/types/feed';
```

Comment:
A new type (`ActivityFilterStatus`) was created to be used with filters because `FeedItemStatus` is used only for comments and annotations to differentiate between open and resolved.